hub/scripts.js

```diff
@@ -245,6 +245,26 @@
     });
   }
 
+  /**
+   * Loads a JS file on the page
+   * @param {String} src The path to the JS file
+   * @param {Object} param The parameters used to load the file
+   */
+  function loadJS(src, param) {
```

Comment:
```suggestion
  function loadJS(src, params) {
```

Reply:
or

```suggestion
  function loadJS(src, id) {
```

if you only support the `id` parameter anyway.